### PR TITLE
Adds drafts of UX guides

### PR DESCRIPTION
**Pull request by dgesin**
Have added a bunch of the new UX guides created by Vianka.  Do not have card graphic yet.  Will add when it is completed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 <div class="guides">
     <section class="wrapper row">
         <div class="section-introduction">
-            <h1>Developer Guides</h1>
+            <h1>Developer Guides Test test</h1>
```
Comment (bmathews) [inline index.html]: "Test test"

File: style-guides.md [hunk 2]
```diff
@@ -1,47 +1,37 @@
...
 
-> "-."
-There are iterations and sign-offs involved to have a final style guide completed but this is can take anywhere from ____________
+For a more holistic technical approach, we include our style guides as part of the [pattern library]({{site.baseurl}}/pattern-library).
 
-## Outcome 
+## How long does this take?
 
-> "-."
+There are iterations and sign-offs involved to have a final style guide completed but this is can take anywhere from (**TODO**)
```
Comment (natoverse) [inline style-guides.md]: I thought it would be better to just write "todo" in the guide than leave an empty placeholder.

File: user-personas.md [hunk 3]
```diff
@@ -1,48 +1,41 @@
...
 * Refine: Combine and prioritize the rough personas. Separate them into primary, secondary, and, if necessary, complementary categories. You should have roughly 3-5 personas and their identified characteristics.
-* Make them realistic: Develop the appropriate descriptions of each personas background, motivations, and expectations. 
-* Have the information found through the user interviews and research direct you in how to create these personas. 
-
-Have any examples of personas created in past projects?
+* Make them realistic: Develop the appropriate descriptions of each personas background, motivations, and expectations.
+* Have the information found through the user interviews and research direct you in how to create these personas.
 
-## Outcome 
+**TODO: include examples of personas created in past projects**
```
Comment (natoverse) [inline user-personas.md]: The previous line here sounded like a placeholder for a todo.

Comment (natoverse) [inline user-personas.md]: (line 39 in the diff)

**Comment by natoverse** [comment]
In a couple of places I cross-linked between guides when something was mentioned. We should keep an eye out for more places to do that, so it is more "surfable" like Wikipedia.

**Pull request merged**